src/slashcommands/SilenceRole.ts

```diff
@@ -0,0 +1,77 @@
+import { ApplicationCommandDataResolvable, CommandInteraction, CacheType, Role, MessageEmbed, Interaction } from "discord.js";
+import { ApplicationCommandOptionTypes } from "discord.js/typings/enums";
+import Enmap from "enmap";
+import { Bot } from "../Bot";
+import { SlashCommand } from "./SlashCommand";
+import { managerRoles } from "./ManagerRole";
+import { silencedUsers } from "./SilenceMember";
+
+export const silencedRole = new Enmap('SilencedRole');
+
+export class SilenceRole implements SlashCommand {
+    name: string = 'silencerole';
+    registerData: ApplicationCommandDataResolvable = {
+        name: this.name,
+        description: '[MANAGER] Set a role from using Birthday, Intro, or Music commands.',
+        options: [{
+            name: 'role',
+            description: 'The role to silence.',
+            type: ApplicationCommandOptionTypes.ROLE,
+            required: true,
+        }]
+    }
+    requiredPermissions: bigint[] = [];
+    run(bot: Bot, interaction: CommandInteraction<CacheType>): Promise<void> {
+        try{
+            //TODO if someone attempts to set the same role that is already set, remove it
```

Comment:
Let's try to tackle TODOs before submitting PRs. There's a draft feature for a reason.

Reply:
resolved this already, forgot to remove the /TODO